src/stylus/fablevision/parent-drake.styl

```diff
@@ -33,3 +33,8 @@
     margin-top: 10px
   .organism
     margin-top: -8px
+
+.os-windows
+  .geniblocks.parentDrake
+    .parentTitle
+      margin-top: 1rem
```

Comment:
Were you able to find any consistent way to determine how much the Windows offset needs to be? Checking for every text component seems tedious